kytos/templates/skel/napp-structure/username/napp/setup.py.template

```diff
@@ -0,0 +1,256 @@
+# Path for enabled NApps
+ENABLED_PATH = VAR_PATH / 'napps'
+# Path to install NApps
+INSTALLED_PATH = VAR_PATH / 'napps' / '.installed'
+CURRENT_DIR = Path('.').resolve()
+
+# NApps enabled by default
+CORE_NAPPS = []
+
+
+class SimpleCommand(Command):
+    """Make Command implementation simpler."""
+
+    user_options = []
+
+    @abstractmethod
+    def run(self):
+        """Run when command is invoked.
+
+        Use *call* instead of *check_call* to ignore failures.
+        """
+
+    def initialize_options(self):
+        """Set default values for options."""
+
+    def finalize_options(self):
+        """Post-process options."""
+
+
+class Cleaner(SimpleCommand):
+    """Custom clean command to tidy up the project root."""
+
+    description = 'clean build, dist, pyc and egg from package and docs'
+
+    def run(self):
+        """Clean build, dist, pyc and egg from package and docs."""
+        call('rm -vrf ./build ./dist ./*.egg-info', shell=True)
+        call('find . -name __pycache__ -type d | xargs rm -rf', shell=True)
+        call('make -C docs/ clean', shell=True)
+
+
+class TestCoverage(SimpleCommand):
+    """Display test coverage."""
+
+    description = 'run unit tests and display code coverage'
+
+    def run(self):
+        """Run unittest quietly and display coverage report."""
+        cmd = 'coverage3 run -m unittest && coverage3 report'
+        call(cmd, shell=True)
+
+
+class Linter(SimpleCommand):
+    """Code linters."""
+
+    description = 'lint Python source code'
+
+    def run(self):
+        """Run yala."""
+        print('Yala is running. It may take several seconds...')
+        check_call('yala *.py', shell=True)
+
+
+class CITest(SimpleCommand):
+    """Run all CI tests."""
+
+    description = 'run all CI tests: unit and doc tests, linter'
+
+    def run(self):
+        """Run unit tests with coverage, doc tests and linter."""
+        cmds = ['python3.6 setup.py ' + cmd
+                for cmd in ('coverage', 'lint')]
+        cmd = ' && '.join(cmds)
+        check_call(cmd, shell=True)
+
+
+class KytosInstall:
+    """Common code for all install types."""
+
+    @staticmethod
+    def enable_core_napps():
+        """Enable a NAPP by creating a symlink."""
+        (ENABLED_PATH / '{{username}}').mkdir(parents=True, exist_ok=True)
+        for napp in CORE_NAPPS:
+            napp_path = Path('{{username}}', napp)
+            src = ENABLED_PATH / napp_path
+            dst = INSTALLED_PATH / napp_path
+            symlink_if_different(src, dst)
+
+
+class EggInfo(egg_info):
+    """Prepare files to be packed."""
+
+    def run(self):
+        """Build css."""
+        # self._install_deps_wheels()
+        super().run()
+
+    # @staticmethod
+    # def _install_deps_wheels():
+    #    """Python wheels are much faster (no compiling)."""
+    #    print('Installing dependencies...')
+    #    check_call([sys.executable, '-m', 'pip', 'install', '-r',
+    #                'requirements/run.in'])
```

Comment:
I think this class is now deprecated. Also, please take a look at kytos/kytos#1114.

Reply:
I think this class can be safely removed